admin_guide/router.adoc

```diff
@@ -54,3 +54,91 @@ connecting to back-ends for re-encrypt terminated routes are stored in the
 namespace and name of the route. The key, certificate, and CA certificate are
 concatenated into a single file. You can use
 link:https://www.openssl.org/[OpenSSL] to view the contents of these files.
+
+== Customizing a Router
+
+The HAProxy router is based on a golang template.  This template generates the
+HAProxy configuration file.  If you would like to customize a router to meet
+your needs you are able to change the template file, build a new docker image,
+and run a customized router.
+
+One common case for this may be implementing new features withing the
+application backends.  For instance, it might be desirable in a highly available
+setup to use stick-tables that synchronizes between peers.  The router
+plugin provides all the facilities necessary to make this customization.
+
+.Using Stick Tables
```

Comment:
It was meant to mark the start of the example customization of the router.  Didn't really think it should be a clickable part of the TOC but I have no preference either way.